src/main/java/no/nav/familie/integrasjoner/tilgangskontroll/CachedTilgangskontrollService.kt

```diff
@@ -19,21 +21,22 @@ class CachedTilgangskontrollService(private val egenAnsattService: EgenAnsattSer
                key = "#jwtToken.subject.concat(#personIdent)",
                condition = "#personIdent != null && #jwtToken.subject != null")
     fun sjekkTilgang(personIdent: String, jwtToken: JwtToken): Tilgang {
-        val personInfo = personopplysningerService.hentPersoninfo(personIdent)
-        val diskresjonskode = personInfo.diskresjonskode
-
-        if (DISKRESJONSKODE_KODE6 == diskresjonskode) {
-            return hentTilgangForRolle(tilgangConfig.grupper["kode6"], jwtToken, personIdent)
-        }
-
-        if (DISKRESJONSKODE_KODE7 == diskresjonskode) {
-            return hentTilgangForRolle(tilgangConfig.grupper["kode7"], jwtToken, personIdent)
-        }
+        val personInfo = personopplysningerService.hentPersoninfo(personIdent, "BAR", PersonInfoQuery.ENKEL)
```

Comment:
Er det noen som har forslag til hvordan vi kan sette riktig tema på en fornuftig måte her? Det er ikke helt rett frem å finne ut hvem som har kalt tilgangskontrollen 😒

Reply:
Burde ABAC bli tatt i bruk istedet for å kalle PDL for tilgangskontroll?
Cached tilgangskontroll, så trenger vi ikke å belaste PDL for dette

Vet ikke hva den tjenesten egentlige heter men den ble nevnt på teknisk demo, https://navno.sharepoint.com/:p:/r/sites/TeknologiradarRedaksjonen/_layouts/15/Doc.aspx?action=edit&sourcedoc=%7B7215414D-84B9-4D39-B7B8-2E0FE7CF2A1D%7D

Reply:
Husker ikke grunnen, men ABAC ble valgt bort.

Hva med å ha med TEMA inn på Tilgangskontrolleren?

Reply:
Vi kan godt sjekke om premissene for å ta i bruk ABAC har endret seg (jeg husker heller ikke hvorfor vi valgte det bort), men det er en større endring som må bli en egen oppgave. Vi må uansett slutte å bruke TPS.